src/Modules/Pages/Domain/Revision/Form/RevisionContentType.php

```diff
@@ -92,7 +92,14 @@ private function buildrevisionBlockForm(
         }
 
         if ($loadDefaultBlocks) {
-            $form->add('blocks_1', TextareaType::class, ['label' => 'lbl.Content', 'required' => false]); // TODO: remove this
+            $form->add(
```

Comment:
I would remove this block for now since it causes issues, and change the todo to implement this

Reply:
fixed 614a203cbbdfd3d8ef2b5b84d0561dd693ea371a